confignode/src/main/java/org/apache/iotdb/confignode/service/thrift/ConfigNodeRPCServiceProcessor.java

```diff
@@ -410,6 +415,25 @@ public TSStatus dropFunction(TDropFunctionReq req) throws TException {
     return configManager.dropFunction(req.getUdfName());
   }
 
+  @Override
+  public TSStatus flush(TFlushReq req) throws TException {
+    IClientManager<TEndPoint, SyncDataNodeInternalServiceClient> INTERNAL_SERVICE_CLIENT_MANAGER =
```

Comment:
There is a SyncDataNodeClientPool in configode.client

Reply:
fixed~